ocs_ci/templates/workloads/files_create_delete.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
please check if this can be converted to python

Reply:
This can be possible. I will check